identity.go

```diff
@@ -59,13 +60,21 @@ func (i Identity) Whois() string {
 }
 
 // WhoisAdmin returns a whois description for admin users.
-func (i Identity) WhoisAdmin() string {
+func (i Identity) WhoisAdmin(room *chat.Room, host *Host) string {
```

Comment:
Hm, I don't mind the feature overall but I kinda wanna avoid changing this function signature. Lemme think on this if I can think of a better way.

Reply:
How about removing the `host *Host` parameter (I don't think we actually need it, but it is correct that op is relative to a room so we do need it).

Then we can do:

```go
if member, ok := room.MemberByID(i.ID()); ok && room.IsOp(member.User) {
	isOp = " > op: true" + message.Newline
}
```

Note I also changed the format of the output to be more consistent with the rest. Also, if you don't mind, could you move it to the last line? I don't want to break any bots that assume the ordering of whois fields.

```